Utils/PluginActivator/source/Log.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
```

Comment:
All functionality realized here is already part of Thunder e.g. LOGGING/TRACING with a distinct different meaning for LOGGING and TRACING through the tracing .lib and R4 messaging lib. Which is already a dependency as COMRPC is dependend on it, so no new additional dependencies.